src/chttpd/src/chttpd_node.erl

```diff
@@ -358,8 +358,13 @@ format_pid_stats(Mailboxes) ->
 get_distribution_stats() ->
     lists:map(
         fun({Node, Socket}) ->
-            {ok, Stats} = inet:getstat(Socket),
-            {Node, {Stats}}
+            try inet:getstat(Socket) of
+                {ok, Stats} ->
+                    {Node, {Stats}}
+            catch
+                _:_ ->
+                    {Node, {}}
```

Comment:
This will crash as `{}` is not valid json. So at least it would have to be `{[]}`

Another thing to consider if anything interpreting the results actually expects all those keys in the json object and cannot handle an empty object. Then it could make sense to emit:

```
{[
     {recv_oct,0},
     {recv_cnt,0},
     {recv_max,0},
     {recv_avg,0},
     {recv_dvi,0},
     {send_oct,0},
     {send_cnt,0},
     {send_max,0},
     {send_avg,0},
     {send_pend,0}
]}
```

We'd have to see if we ever documented the response type as having those keys or not. If not an empty json object is probably ok

Reply:
Even if we do not promise to return these keys, I still think it is better to return them anyways instead of `{[]}`.

Reply:
Probably safer, yeah

Reply:
Added.

Reply:
Looks good! Don't forget to mark the PR as ready for review and rebase on top of latest master